src/Fantomas.Tests/MultilineBlockBracketsOnSameColumnRecordTests.fs

```diff
@@ -0,0 +1,438 @@
+module Fantomas.Tests.MultilineBlockBracketsOnSameColumnRecordTests
+
+open NUnit.Framework
+open FsUnit
+open Fantomas.Tests.TestHelper
+
+let config = ({ config with
+                    MultilineBlockBracketsOnSameColumn = true
+                    SpaceBeforeColon = true
+                    SpaceBeforeSemicolon = true })
+
+[<Test>]
+let ``single member record stays on oneline`` () =
```

Comment:
I think there should be a space between one and line

Reply:
Indeed, thanks.